pkg/cloud/aws/services/ec2/routetables_test.go

```diff
@@ -118,8 +117,8 @@ func TestReconcileRouteTables(t *testing.T) {
 				VPC: v1alpha1.VPCSpec{
 					InternetGatewayID: aws.String("igw-01"),
 					ID:                "vpc-routetables",
-					Tags: tags.Map{
-						tags.ClusterKey("test-cluster"): "owned",
+					Tags: v1alpha1.Map{
+						v1alpha1.ClusterKey("test-cluster"): "owned",
```

Comment:
We should prefix the exposed functions as well, reading this line without going to the definition is hard to understand this is meant to be used in tags.